### ISSUE TITLE
Tokens in emails

### ISSUE THREAD

**Issue by bobintetley** [bobintetley/asm3#397]
One customer would like wordkey tokens (eg: AnimalName, OwnerName) to be usable in email bodies when sending documents to people from the media tab of records. This becomes more useful with #502 since tokens can be included in standard template emails.

**Comment by bobintetley** [comment]
This was already done shortly after #502
